packages/babel-helper-wrap-function/src/index.js

```diff
@@ -12,7 +12,7 @@ const buildWrapper = template(`
   })
 `);
 
-const namedBuildWrapper = template(`
+const buildNamedExpresionWrapper = template.expression(`
```

Comment:
`buildNamedExpresionWrapper` `->` `buildnamedExpressionWrapper`

Reply:
why would you rename this? casing in `buildNamedExpresionWrapper` seems fine to me

Reply:
I forgot an `s`

Reply:
Ha yes, of course I make a separate mistake when pointing out the typo though 🤦‍♂️